test/views/SoftwareChoicesFilterViewSpec.scala

```diff
@@ -175,10 +178,25 @@ class SoftwareChoicesFilterViewSpec extends ViewBaseSpec with SoftwareProvidersT
         }
       }
 
+      "contain a filter for Type of software" should {
+
+        "Suitable for should contain Type of software filter header" in {
+          document.select(Selectors.filterHeader(2)).text shouldBe FilterSearchMessages.typeOfSoftware
+        }
+
+        "Filter Header contains Accounting software" in {
+          document.select(Selectors.accountingFilter).text shouldBe FilterSearchMessages.accountingSoftware
+        }
+
+        "Filter Header contains Connects to spreadsheets" in {
+          document.select(Selectors.spreadsheetsFilter).text shouldBe FilterSearchMessages.spreadSheets
+        }
+      }
+
       "contain a filter for Additional Features" should {
 
         "Suitable for should contain Additional software features filter header" in {
-          document.select(Selectors.filterHeader(2)).text shouldBe FilterSearchMessages.additionalSoftwareFeatures
+          document.select(Selectors.filterHeader(3)).text shouldBe FilterSearchMessages.additionalSoftwareFeatures
```

Comment:
I know this isn't something you added, but I'd warn against using an index to select the right filter header. They could get out of step and end up not testing the correct thing - there's no guarantee on the left hand side of the assertion statement that we're checking the right one. 

I'd suggest updating the filter helper to add a unique ID to each filter section & then document.select to explicitly grab that ID.

i.e. if the ID was "price-filter", in the test you'd write `document.select("price-filter` h2").text shouldBe ... ` Actually, you could stick the selector in a constant like you have with the rest to be consistent instead, e.g. `document.select(Selectors.priceFilterHeader).text shouldBe ...`